torch/_torch_docs.py

```diff
@@ -1502,6 +1502,63 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.clamp`.
 """.format(**common_args))
 
+add_docstr(torch.complex,
+           r"""
+complex(real, imag, *, out=None) -> Tensor
+
+Constructs a complex tensor with its real part equal to :attr:`real` and its
+imaginary part equal to :attr:`imag`.
+
+Args:
+    real (Tensor): The real part of the complex tensor. Must be float or double.
+    imag (Tensor): The imaginary part of the complex tensor. Must be same dtype
+        as :attr:`real`.
+
+Keyword args:
+    {out} If the inputs are ``torch.float32``, must be ``torch.complex64``. If
```

Comment:
"{out} -> out (Tensor): "

You don't want to autogenerate the out doc, which is what happens when using `{out}`